package.json

```diff
@@ -1205,6 +1205,96 @@
           ],
           "scope": "window"
         },
+        "cmake.statusbar.kit.type": {
+          "type": "string",
+          "default": "text",
+          "enum": [
+            "text",
+            "short",
+            "hidden"
+          ],
+          "scope": "window"
+        },
+        "cmake.statusbar.kit.length": {
+          "type": "integer",
+          "default": 10,
```

Comment:
10 is really short and probably won't display most Kits in a manner that is helpful. I'm not convinced that this is a feature anyone will use. I would be happy with "text", "icon", and "hidden" though I don't see an "icon" option for this one. Since this aligns with the toolset that you'll be using to compile your code, I would vote for the $(tools) or $(briefcase) octicons to represent kits.

Reply:
increased to 20 and added option for icon (tools).